Android/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+cmake_minimum_required (VERSION 2.8.7)
+project(MCServer)
+include(ExternalProject)
+
+externalproject_add(lua_native
+	SOURCE_DIR ../lib/lua
+	CMAKE_GENERATOR "Visual Studio 14 2015"
```

Comment:
Would anyone have any idea about this? It could be a showstopper for integration into @bearbin's Jenkins server.